ui/app/components/secret-edit-metadata.js

```diff
@@ -54,6 +54,7 @@ export default class SecretEditMetadata extends Component {
     if (value) {
       if (name === 'customMetadata') {
         // cp validations won't work on an object so performing validations here
+        // TODO: review this and incorporate into model-validations system
```

Comment:
Can you put your initials in front of the TODO? e.g. ARG TODO or CBS TODO (you'll see those examples in front of some in the codebase). This way you can search for your specific TODOs

Reply:
For sure thanks for the tip!